integrations/airtable/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "target": "es2017",
+    "baseUrl": ".",
+    "outDir": "dist",
+    "checkJs": false
+  },
+  "include": [".botpress/**/*", "src/**/*", "package.json"]
```

Comment:
Query: Do we need to include package.json here ?

Reply:
Yo @subashcs, this is a tsconfig that copy in all integrations. In some of them, we import the package.json file to read the version